.travis.yml

```diff
@@ -15,6 +15,8 @@ before_script:
 script:
   - docker-compose run app coverage run -a --source=pombola,pombola_sayit,wordcloud,writeinpublic ./manage.py test --settings=pombola.settings.tests_south_africa
   - curl --connect-timeout 20 --retry 5 --retry-delay 1 http://localhost:8000/position/member/parliament/\?order\=name\&a\=1 | grep "Ramokgopa"
+  - curl --connect-timeout 20 --retry 5 --retry-delay 1 http://localhost:8000/committees/ | grep "26 Nov 2020 | Became"
+  - curl --connect-timeout 20 --retry 5 --retry-delay 1 http://localhost:8000/organisation/committee-multi-party-womens-caucus/ | grep "26 Nov 2020 | Became"
```

Comment:
We don't want more of these tests - the curl check above is just to be sure that the demo data vaguely works - you need to test the views in the way we normally test view code - by setting up data specific to what you're testing in that view